third_party/airflow/armada/operators/armada.py

```diff
@@ -29,6 +29,8 @@
 
 armada_logger = logging.getLogger("airflow.task")
 
+ANNOTATION_TASK_ID = "armadaproject.io/taskId"
```

Comment:
This is actually configurable from lookout so we should look more into this. Env variable or read from lookout confit.

Reply:
No, I mean that lookout will decide what annotations to show up if that match the part before '/'.  

https://github.com/G-Research/armada/blob/master/config/lookout/config.yaml#L16

So annotations that are displayed to user in lookout have to have a form that is present in the config.  Otherwise, we ignore them.  

The pod object has to add the annotation that matches how we deploy lookout to display annotations.  We should probably see if this is really just a constant value or changes depending on production environment in GR.

Reply:
it doesn't seem like airflow module should pull in lookout config